.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: Auto Assign to Partner Project
+on:
+  issues:
+    types: [opened, labeled]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to Partner Project
+    steps:
+    - name: Assign NEW issues Partner Project
```

Comment:
This will assign all new issues and PRs of the repo to this project, is this really what you want?